website/docs/for-capacitor/overview.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Overview
+sidebar_label: Overview
+---
+
+## Why use micro-frontends
```

Comment:
I feel like we might want a short paragraph bridging the differences between the two different Portals here. I was thinking something like:

`Portals for Capacitor is a flavor of Portals that works within Ionic Capacitor (link)...`